.github/workflows/CI.yml

```diff
@@ -0,0 +1,49 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  CI:
+    strategy:
+      matrix:
+        backend: ["OPENMP"]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install deps
+        run: sudo apt-get install gfortran openmpi-bin libopenmpi-dev
+      - name: Checkout code
+        uses: actions/checkout@v2.2.0
+      - name: Checkout kokkos
+        uses: actions/checkout@v2.2.0
+        with:
+          repository: kokkos/kokkos
+          ref: 3.1.00
+          path: kokkos
+      - name: Checkout Cabana
```

Comment:
Maybe we should re-order the checkout and build blocks for better readability.

Reply:
Agreed - Kokkos, then Cabana

Reply:
Done.